src/css/style.css

```diff
@@ -329,51 +328,47 @@ button {
     background-color: #ffffff;
 }
 .gpii-fd-navButtons-back, .gpii-fd-navButtons-next {
-    border: 0.2rem solid #fff;
-    border-radius: 0.4rem;
+    border-radius: 0.5rem;
     visibility: hidden;
-    margin: 1rem;
-    padding: 0.5rem 1.5rem;
-    line-height: 4.2rem;
-}
-
-.gpii-fd-navButtons-back:before,
-.gpii-fd-navButtons-next:after {
     background-color: #425165; /* blue-grey */
     color: #ffffff;
-    border-radius: 50rem;
-    position: relative;
-    font-family: "PGA-FD";
-    font-size: 4rem;
-    top: 0.7rem;
+    padding: 1rem;
+    line-height: 2rem;
+    height: 4.2rem;
+}
+.gpii-fd-navButtons-back {
+    padding-left: 0;
+    margin: 0 0 1rem 1rem;
+}
+.gpii-fd-navButtons-next {
+    padding-right: 0;
+    margin: 0 1rem 1rem 0;
 }
 .gpii-fd-navButtons-back:hover,
 .gpii-fd-navButtons-next:hover {
-    color: inherit; /* to override foundation's blue */
-    background-color: inherit;
-}
-.gpii-fd-navButtons-back:hover:before,
-.gpii-fd-navButtons-next:hover:after {
-    box-shadow: 0 0 0 0.2rem rgb(255, 255, 255), 0 0 0 0.6rem rgb(72, 82, 100); /*blue-grey */
+    background-color: #425165; /* blue-grey, to override foundation's blue */
+    box-shadow: 0 0 0 0.1rem rgb(255,255,255), /* white */
+                0 0 0 0.6rem rgb(72, 82, 100); /*blue-grey */
 }
 .gpii-fd-navButtons-back:focus,
 .gpii-fd-navButtons-next:focus {
-    color: #425165;
+    background-color: #425165; /* blue-grey */
+    box-shadow: 0 0 0 0.4rem rgb(244, 215, 75), /* Yellow */
```

Comment:
It seems the yellow outline from the focus styling doesn't work in Safari. Is that a known issue?